crates/ruff_linter/src/rules/flake8_commas/rules/trailing_commas.rs

```diff
@@ -29,22 +30,33 @@ enum TokenType {
 
 /// Simplified token specialized for the task.
 #[derive(Copy, Clone)]
-struct Token<'tok> {
-    type_: TokenType,
-    // Underlying token.
-    spanned: Option<&'tok Spanned>,
+struct Token {
+    r#type: TokenType,
+    range: TextRange,
```

Comment:
The raw token was never used, so let's simplify this a bit.

I've also renamed `type_` to `r#type`. This is just a personal preference and I can revert if the former is better.